services/static-webserver/client/source/class/osparc/dashboard/NewPlusMenu.js

```diff
@@ -200,55 +237,55 @@ qx.Class.define("osparc.dashboard.NewPlusMenu", {
     },
 
     __addDisabledButton: function(newStudyData) {
-      const menuButton = this.self().createMenuButton(null, newStudyData.title, newStudyData.reason);
-      osparc.utils.Utils.setIdToWidget(menuButton, newStudyData.idToWidget);
+      const menuButton = this.self().createMenuButton(null, newStudyData["title"], newStudyData["reason"]);
```

Comment:
I wonder why `newStudyData.idToWidget` now needs dict-like access `newStudyData["idToWidget"]`
is is something id did in the api response?

Reply:
it's the same